ironfish/src/wallet/account.test.ts

```diff
@@ -1834,25 +1834,68 @@ describe('Accounts', () => {
       await node.wallet.updateHead()
 
       let unspentHashes = await AsyncUtils.materialize(
-        accountA['walletDb'].loadUnspentNoteHashes(accountA),
+        accountA['walletDb'].loadUnspentNoteHashes(accountA, Asset.nativeId()),
       )
       expect(unspentHashes).toHaveLength(1)
       const unspentHash = unspentHashes[0]
 
       const transaction = await useTxFixture(node.wallet, accountA, accountA)
 
       unspentHashes = await AsyncUtils.materialize(
-        accountA['walletDb'].loadUnspentNoteHashes(accountA),
+        accountA['walletDb'].loadUnspentNoteHashes(accountA, Asset.nativeId()),
       )
       expect(unspentHashes).toHaveLength(0)
 
       await accountA.expireTransaction(transaction)
 
       unspentHashes = await AsyncUtils.materialize(
-        accountA['walletDb'].loadUnspentNoteHashes(accountA),
+        accountA['walletDb'].loadUnspentNoteHashes(accountA, Asset.nativeId()),
       )
       expect(unspentHashes).toHaveLength(1)
       expect(unspentHash).toEqualBuffer(unspentHashes[0])
     })
   })
+
+  describe('getUnspentNotes', () => {
```

Comment:
I'm not sure how inconvenient it is to test, but maybe a test that tests filtering by asset ID could be useful?

Reply:
Not too inconvenient at all! I added a test of filtering with multiple assets.